### ISSUE TITLE
Search index of docs contains links to older versions

### ISSUE THREAD

**Issue by Philippus** [akka/akka-http#1587]
It looks like the search index contains links to older versions of the docs.
If I search in the docs for 'Marshalling' (as an example) the suggestion box contains links to older versions of the docs.

<img width="519" alt="schermafdruk 2017-12-02 09 38 41" src="https://user-images.githubusercontent.com/1923596/33513517-d0720378-d744-11e7-9f3e-19252f4d1e44.png">

**Comment by jrudolph** [comment]
Thanks for the report. Indeed, it should only return results for the latest version.

**Comment by ktoso** [comment]
Seems the current query code is not using the version property? `view-source:https://doc.akka.io/docs/akka/current/stream/`

```

inputSelector: '#overlay-search',
algoliaOptions: {
hitsPerPage: 5,
facetFilters: ["language:" + lang]
}
```

with the old: `view-source:https://doc.akka.io/docs/akka/2.4.2/`

```
    algoliaOptions: {
      hitsPerPage: 10,
      facetFilters: '["language:' + lang + '","version:' + version + '"]'
```

**Comment by ktoso** [comment]
So this seems like a bug to me when we moved to paradox perhaps?

**Comment by jrudolph** [comment]
No, it's here in akka-http and only since this version. (But it might still be related to our paradox plugin).

**Comment by ktoso** [comment]
I see -- it breaks when it's on "`/current/`", while it works on a specific version page.
We should make the `/current/` one work by figuring out what the version of those docs is (locateable in the HTML)

**Comment by jrudolph** [comment]
It seems to be broken or https://doc.akka.io/docs/akka-http/10.0.11/ as well

**Comment by jrudolph** [comment]
See https://github.com/akka/akka-paradox/pull/9, the filter was removed for some undocumented reason...

**Comment by jrudolph** [comment]
Preliminary fixed in #1833.

**Comment by jrudolph** [comment]
This was now fixed by reverting my previous fix and reconfiguring algolia to only index the `current` version of docs here: https://github.com/algolia/docsearch-configs/pull/308

The conclusion is that all search will end up on the current version even when browsing older ones. We think that is reasonable behavior because we want to deemphasize old documentation anyway and keeping those pages indexed might be hard anyway.

Thanks @2m for doing the final push.
